Test/civl/inductive-sequentialization/2PC.bpl

```diff
@@ -77,20 +75,18 @@ INV4 ({:linear_in "pid"} pids:[int]bool)
 returns ({:pending_async "PARTICIPANT2"} PAs:[PA]int, {:choice} choice:PA)
 modifies ReqCH, VoteCH, DecCH, votes, decisions;
 {
-  assert Init(pids, ReqCH, VoteCH, DecCH, decisions);
-
-  havoc ReqCH, VoteCH, DecCH, votes, decisions;
+  var {:pool "INV1"} k: int;
 
+  assert Init(pids, ReqCH, VoteCH, DecCH, decisions);
+  assume {:add_to_pool "INV1", k, k+1} {:add_to_pool "PARTICIPANT2", PARTICIPANT2(n)} 0 <= k && k <= n;
```

Comment:
```suggestion
  assume
    {:add_to_pool "INV1", k, k+1}
    {:add_to_pool "PARTICIPANT2", PARTICIPANT2(n)}
    0 <= k && k <= n;
```

I find it more readable like this (and similar in other places). What do you think?